src/Cli/ConfigGenerator.cs

```diff
@@ -681,6 +686,99 @@ private static bool TryUpdateDepthLimit(
             return true;
         }
 
+        /// <summary>
+        /// Attempts to update the Config parameters in the GraphQL runtime settings based on the provided value.
+        /// Performs the update on the runtimeConfig which is passed as reference
+        /// Returns true if the update has been performed, else false
+        /// </summary>
+        /// <param name="options">Options including the graphql runtime parameters.</param>
+        /// <param name="runtimeConfig">Current config, updated if method succeeds.</param>
+        /// <returns>True if the update was successful, false otherwise.</returns>
+        private static bool TryConfigureRuntimeOptions(
+            ConfigureOptions options,
+            [NotNullWhen(true)] ref RuntimeConfig runtimeConfig)
+        {
+            // GraphQL: Enabled, Path, Allow-Introspection and Multiple-Mutations.Create.Enabled
+            GraphQLRuntimeOptions? updatedGraphQLOptions = runtimeConfig?.Runtime?.GraphQL;
+            if (options.RuntimeGraphQLEnabled != null ||
+                options.RuntimeGraphQLPath != null ||
+                options.RuntimeGraphQLAllowIntrospection != null ||
+                options.RuntimeGraphQLMultipleMutationsCreateEnabled != null)
+            {
+                bool status = UpdateConfigureGraphQLValues(options, ref updatedGraphQLOptions);
+                if (status)
+                {
+                    runtimeConfig = runtimeConfig! with { Runtime = runtimeConfig.Runtime! with { GraphQL = updatedGraphQLOptions } };
+                }
+                else
+                {
+                    return false;
+                }
+            }
+
+            return runtimeConfig != null;
+        }
+
+        /// <summary>
+        /// Attempts to update the Config parameters in the GraphQL runtime settings based on the provided value.
+        /// Validates that any user-provided parameter value is valid and then returns true if the updated GraphQL options
+        /// needs to be overwritten on the existing config parameters
+        /// </summary>
+        /// <param name="options">options.</param>
+        /// <param name="updatedGraphQLOptions">updatedGraphQLOptions.</param>
+        /// <returns>True if the value needs to be udpated in the runtime config, else false</returns>
+        private static bool UpdateConfigureGraphQLValues(
+            ConfigureOptions options,
```

Comment:
nit: since this function is "attempting", consider naming it as TryXXX

Reply:
updated